recipe/meta.yaml

```diff
@@ -44,7 +35,7 @@ requirements:
     - clangxx {{ llvm_version }}.*
     - llvm-tools {{ llvm_version }}.*
   run:
-    - python *
+    - python
```

Comment:
Why are you removing this? This is not desirable. In fact, we want to keep it but probably do #39 first.